docs/upgrading.md

```diff
@@ -0,0 +1,13 @@
+# Upgrading your installed application
+
+Upgrading the application is **not supported**.
+
+Fleet UI does not allow to upgrade an Elastic Agent running under Kubernetes, as that would conflict with the orchestration performed by Kubernetes itself.
+
+GCP Marketplace does not support upgrading an application via its UI.
+
+## Workaround
+
+To workaround this there are 2 ways:
+1. once installed the Application behaves like any other Kubernetes resource. You can edit the application manifest (as you usually would with other Kubernetes resources). To do this, please refer to the Elastic Agent on Kubernetes upgrade documentation;
```

Comment:
can we link to the referenced docs?

Reply:
updated in [`de3efcd` (#1)](https://github.com/elastic/elastic-agent-gcp-kubernetes-app/pull/1/commits/de3efcdea01e14f062af2ac990c9bb4a184820d6)